app/javascript/application.js

```diff
@@ -1,6 +1,7 @@
 import $ from 'jquery';
 import _ from 'underscore';
 import Rails from '@rails/ujs';
+export const Clipboard = require("clipboard");
```

Comment:
```suggestion
import Clipboard from 'clipboard';
```

This works for me, similar to the other imports.